public/js/maps.js

```diff
@@ -1,83 +1,53 @@
 (function() {
-  const mymap = L.map('mapid').setView([52.3754419819, 4.930586814], 13);
-  const marker = L.marker();
+  const mymap = (window.mymap = L.map('map').setView([52.378663542, 4.9005202576], 14));
   const circle = new L.circle();
 
-  function formatter(num) {
-    return num > 999 ? (num / 1000).toFixed(0) + " km" : num + " m";
-  }
-
   L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
-    attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
     maxZoom: 18,
-    id: 'mapbox.streets',
+    id: 'mapbox.streets-satellite',
     accessToken: 'pk.eyJ1IjoidGhvbWFzb2ZmaW5nYSIsImEiOiJjamc3cTlyczgwdWs0MndrbDhmeWdrd3RkIn0.44lDtgLNa9soBfbU67ASpQ'
   }).addTo(mymap);
 
-  // Add some map controls
-  mymap.addControl(
-    L.control.locate({
-      locateOptions: {
-        maxZoom: 14,
-        drawCircle: false
-      }
-    })
-  );
-  L.Control.geocoder().addTo(mymap);
-
   // Our list of range selectable values
-  let radius = 500; // sets default radius
-  const availableRadius = [50, 100, 500, 1000, 5000, 20000, 50000, 100000, 250000, 500000];
+  let radius = 400; // sets default radius
+  const availableRadius = [50, 100, 200, 400, 600, 800, 1000, 5000, 20000, 50000, 100000, 250000, 500000];
 
   // Form elements
   const radiusInput = document.getElementById('radius');
   const latitudeInput = document.getElementById('latitude');
   const longitudeInput = document.getElementById('longitude');
+  const fileInput = document.getElementById('files');
+  const rangeContainer = document.querySelector('.range');
   const range = document.getElementById('range-input');
-  const output = document.getElementById('range-output');
 
-  output.innerHTML = formatter(radius);
+  if (!range) {
+    return;
+  }
+
+  fileInput.addEventListener('input', (event) => {
+    document.querySelector('.files').classList.add('done');
```

Comment:
As a general rule, don't search for DOM element inside callbacks and looks, it's quite expensive.